README.md

```diff
@@ -35,16 +42,6 @@ Available targets are described in the README in the `cedar-drt` directory.
 
 Additional commands available with `cargo fuzz help`.
 
-## Checking Proof Stability
-
-You can measure the complexity of Dafny proofs using [dafny-reportgenerator](https://github.com/dafny-lang/dafny-reportgenerator/).
-For example, the commands below check that all proofs have a [resource count](https://dafny.org/dafny/VerificationOptimization/VerificationOptimization#identifying-difficult-assertions) under 10M, which is our informal threshold for when a proof is "too expensive" and likely to break with future changes to Dafny and/or Z3.
-
-```bash
-cd cedar-dafny && make verify GEN_STATS=1
-dotnet tool run dafny-reportgenerator summarize-csv-results --max-resource-count 10000000 .
-```
-
```

Comment:
is this section being removed because the tooling no longer works?  or because we are no longer supporting the Dafny?  If the latter, we need to make larger changes to the README (which still mentions the Dafny in many places)

Reply:
It still works, it's just no longer as important because the Dafny will (eventually) be deleted.

Reply:
should we keep this text until we fully remove the Dafny then?  (or at least until we fully remove the Dafny from the README)